src/glove.c

```diff
@@ -21,25 +21,12 @@
 //    GlobalVectors@googlegroups.com
 //    http://nlp.stanford.edu/projects/glove/
 
-
-#include <stdint.h>
```

Comment:
i would leave the includes even when they are also included from common.h.  Otherwise don't we start to get potentially strange results for example with malloc if common.h drops stdlib?

Reply:
It's been forever since I did any serious C programming so I am definitely not an authority on safe C include practices

Reply:
Good point, you are right! :) I didn't notice that you push your changes and I updated my branch. 
https://github.com/przemb/GloVe/commit/ce3a9350b07d87aee7afdba995625e4d6bc5b0a7
If I can only suggest something now - IMO in common.h there is no need for any of 3 mentioned inludes. (but they should be placed to common.c and in all *.c files)